embedded/appendable/multiapp/multi_app_test.go

```diff
@@ -165,8 +171,11 @@ func TestMultiAppClosedFiles(t *testing.T) {
 }
 
 func TestMultiAppReOpening(t *testing.T) {
-	a, err := Open("testdata", DefaultOptions().WithFileSize(1))
-	defer os.RemoveAll("testdata")
+	path, err := ioutil.TempDir(os.TempDir(), "testdata")
+	require.NoError(t, err)
+	defer os.RemoveAll(path)
+
+	a, err := Open(path, DefaultOptions().WithFileSize(1))
```

Comment:
`testdata_copy` below should also be a temp folder

Reply:
ack